README.md

```diff
@@ -1,2 +1,22 @@
 # couchdb
 Minimalist CouchDB client
+
+## Understanding.
+
+client.go --> represents the couchdb client for the others to use.
```

Comment:
Might want to make this into a bulleted list. Github just smashes consecutive lines into  a paragraph (i know, its silly)

```

- client.go - represents the couchdb client for the others to use.
- db.go - This is the low level db interface for the object. (Couch works on rest.)
- dbrequests - Wrapper for all requests from couch.
- view.go - Deals with creation of views.
- document.go - Deals with the creation and updating of documents in the db.
```